sklearn/tests/test_naive_bayes.py

```diff
@@ -34,24 +34,25 @@
 X2 = rng.randint(5, size=(6, 100))
 y2 = np.array([1, 1, 2, 2, 3, 3])
 
+# Test not needed if we remove the classes parameter in partial fit
+# def test_gnb():
```

Comment:
this test is no more required because partial_fit will not have a classes argument. if this works, i'll remove this in next commit. just commented for now.